ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-ref.yaml

```diff
@@ -1,6 +1,6 @@
 ref:
   as: baremetalds-packet-setup
-  from: upi-installer
+  from: devscripts
```

Comment:
Is a dash not allowed? It should have the same name as the git repo

Reply:
I think that the image name is not constrained, we can put the dash